updater/lib/github_api/dependency_submission.rb

```diff
@@ -69,7 +70,7 @@ def payload
         },
         detector: {
           name: SNAPSHOT_DETECTOR_NAME,
-          version: Dependabot::VERSION,
+          version:  Dependabot::Environment.updater_sha || Dependabot::VERSION,
```

Comment:
Could concat the sha onto the version, so we get a bit easier visibility into what version was roughly used, but also 🤷

Reply:
Yeah, that' a fair point

Reply:
https://github.com/dependabot/dependabot-core/pull/13447/commits/9dfc40286d0d846ac97adbe2d399e6d83d21f4f7